src/components/FicheFilm/FicheFilm.jsx

```diff
@@ -74,42 +97,82 @@ export default function FicheFilm() {
     UsersScorePictures = star1;
   }
 
-  const hour = (runtimes - (runtimes % 60)) / 60;
-  const min = (runtimes / 60 - hour) * 60;
+  const Runtime = () => {
+    if (runtimes > 59) {
+      const hour = (runtimes - (runtimes % 60)) / 60;
+      const min = Math.round((runtimes / 60 - hour) * 60 * 100) / 100;
```

Comment:
`min = runtime % 60`, and I'm almost sure I already checked this code on another PR